mir-ci/mir_ci/wayland/virtual_pointer.py

```diff
@@ -53,9 +54,10 @@ def disconnected(self) -> None:
         pass
 
     def xdg_output_logical_size(self, xdg_output, width: int, height: int) -> None:
+        # Whenever the logical size changes, we update the cached physical size
```

Comment:
We should probably just replace this by [wl_output::geometry](https://wayland.app/protocols/wayland#wl_output:event:geometry), we can then avoid thinking of scale here altogether.

But that's for a separate PR.